linode/objects/filtering.py

```diff
@@ -1,12 +1,125 @@
+"""
+.. module:: linode
+
+Collections returned by the :any:`LinodeClient` can be filtered using a
+SQLAlchemy-like syntax.  When calling any "get" method of the :any:`LinodeClient`
+class of one of its groups, any number of filters may be passed in as boolean
+comparisons between attributes of the model returned by the collection.
+
+For example, calling :any:`get_instances` returns a list of :any:`Linode`
+objects, so we can use properties of :any:`Linode` to filter the results::
+
+   # returns all Linodes in the "prod" group
+   client.linode.get_instances(Linode.group == "prod")
+
+You can use any boolean comparisons when filtering collections::
+
+   # returns all Linodes _not_ in us-east-1a
+   client.linode.get_instances(Linode.region != "us-east-1a")
```

Comment:
This one and all the ones below should probably say us-east at this point?

Or are you waiting until next week to be sure?